CondFormats/CTPPSReadoutObjects/plugins/TotemDAQMappingESSourceXML.cc

```diff
@@ -233,23 +234,24 @@ void TotemDAQMappingESSourceXML::setIntervalFor(const edm::eventsetup::EventSetu
   {
     const auto &bl = configuration[idx];
 
+    edm::EventRange range = bl.validityRange;
+
     // event id "1:min" has a special meaning and is translated to a truly minimal event id (1:0:0)
-    EventID startEventID = bl.validityRange.startEventID();
-    if (startEventID.event() == 1)
-      startEventID = EventID(startEventID.run(), startEventID.luminosityBlock(), 0);
+    if (range.startEventID()==edm::EventID(1, 0, 1))
+      range = edm::EventRange(edm::EventID(1, 0, 0), range.endEventID());
```

Comment:
Hi @ghellwig,
It appears that the nomenclature `<run_number>:min` used for the run ranges in the python configuration refers to the event `<run_number>:0:1` instead of `<run_number>:0:0` (first event effectively observed in the stream), thus preventing any channels mapping to be associated for this first, boundary, event.

Again, this is a temporary workaround while a frontier conditions-based mapping is being developed!